Tools/ardupilotwaf/boards.py

```diff
@@ -17,6 +17,9 @@ def __init__(cls, name, bases, dct):
         if cls.abstract:
             return
 
+        if 'toolchain' not in cls.__dict__:
```

Comment:
We should use `if not hasattr(cls, 'toolchain'):`, so that we allow use of inheritance.